.circleci/config.yml

```diff
@@ -1,29 +1,51 @@
 version: 2
 jobs:
+  integration-php72-magento23:
```

Comment:
overall i feel like we should share the steps between unit and integration. the only difference is the script that i s run, we can make a command that has enum as params. lets chat